packages/hdwallet-core/src/transport.ts

```diff
@@ -15,7 +15,7 @@ export abstract class Transport extends eventemitter2.EventEmitter2 {
   /**
    * Must emit outgoing message events and communicate with underlying interface
    */
-  public abstract call(...args: any[]): Promise<any>;
+  public abstract call(...args: any): Promise<unknown>;
```

Comment:
There is a subtle difference -- `any[]` is an array _of a homogenous type_. For example, you can assign a `[string, number]` to `any[]`, but only by narrowing it to `(string|number)[]`, not all the way down to `[string, number]` or `[string, number , ...any[]]` like you might expect. The standard library uses `any` instead of `any[]` for stuff like `Parameters<T>` for this reason.